src/processing/evm.rs

```diff
@@ -0,0 +1,198 @@
+use std::sync::Arc;
+use std::time::{SystemTime, UNIX_EPOCH};
+
+use alloy::hex::FromHex;
+use alloy::primitives::FixedBytes;
+use arc_swap::Guard;
+use bigdecimal::ToPrimitive;
+use num_bigint::BigInt;
+use starknet::core::types::{BlockId, BlockTag, Felt, FunctionCall};
+use starknet::macros::selector;
+use starknet::providers::jsonrpc::HttpTransport;
+use starknet::providers::{JsonRpcClient, Provider};
+
+use crate::config::{Config, EvmConfig};
+use crate::constants::EVM_TIME_SINCE_LAST_FEED_UPDATE;
+use crate::{config::get_config, error::MonitoringError};
+
+#[derive(Clone)]
+enum FeedType {
```

Comment:
those types should be in some `types/cairo` module, smth like that

Reply:
done in cairo/types.rs